From ed35f944b94336019a445460bf51b3ba042bd69c Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Mon, 20 Dec 2021 20:49:15 +0800
Subject: [PATCH 16/16] Revert "drm/sun4i: fix unmet dependency on
 RESET_CONTROLLER for PHY_SUN6I_MIPI_DPHY"

This reverts commit c386d7aa59f5040dbcbc02499a31474859546d6d.
---
 drivers/gpu/drm/sun4i/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/sun4i/Kconfig b/drivers/gpu/drm/sun4i/Kconfig
index 8c796de53222..5755f0432e77 100644
--- a/drivers/gpu/drm/sun4i/Kconfig
+++ b/drivers/gpu/drm/sun4i/Kconfig
@@ -46,7 +46,6 @@ config DRM_SUN6I_DSI
 	default MACH_SUN8I
 	select CRC_CCITT
 	select DRM_MIPI_DSI
-	select RESET_CONTROLLER
 	select PHY_SUN6I_MIPI_DPHY
 	help
 	  Choose this option if you want have an Allwinner SoC with
-- 
2.27.0

